devops/openshift/database-backup/backup-deploy.yaml

```diff
@@ -170,16 +169,16 @@ objects:
                   value: ${MONGODB_AUTHENTICATION_DATABASE}
                 - name: TABLE_SCHEMA
                   value: ${TABLE_SCHEMA}
-                - name: ${DATABASE_SERVER_NAME}_USER
+                - name: DATABASE_USER
```

Comment:
Are we removing the "_USER" because we will no longer support the backup for multiple DBs?
"_USER" reference: https://github.com/BCDevOps/backup-container/tree/master#backupconf

Reply:
Should we decide to move away from back-up of multiple db in same backup container?

If there is a requirement in future to have a another db, or we if decide to back up redis(by any chance) current way will be easily adaptable right?